src/features/user/UserProfile/components/RedeemModal.tsx

```diff
@@ -119,6 +119,8 @@ export default function RedeemModal({
           setCodeRedeemed(true);
           setIsUploadingData(false);
           setCodeValidated(false);
+          const newUser = {...user, score:{personal: res.schemata.treecounter.countPlanted, received: user.score.received}}
```

Comment:
I assume you should use countPersonal here. And we should somehow still show the goal.